build.yaml

```
@@ -5,7 +5,7 @@ buildTimeoutUnit: HOURS
 additionalTestConfigs:
   jdk11:
     testJdkTool: OPEN-JDK11
-    mavenAdditionalArgs: -Dtest=none -DfailIfNoTests=false
+    mavenAdditionalArgs: -Dtest=none -DfailIfNoTests=false -DruntimeVersion=4.8.4
```

If the builds fails remember to add:
```suggestion
    mavenAdditionalArgs: -Dtest=none -DfailIfNoTests=false -DruntimeVersion=4.8.4 -DruntimeProduct=MULE_EE
```